packages/lodestar-utils/src/bytes.ts

```diff
@@ -35,14 +34,6 @@ export function bytesToBigInt(value: Uint8Array, endianness: Endianness = "le"):
   throw new Error("endianness must be either 'le' or 'be'");
 }
 
-export function toHex(buffer: ArrayLike<number>): string {
```

Comment:
Can you put this method back. We should use this method instead of Buffer.from().toHex().

This change doesn't belong in this PR anyways